src/models/scope.ts

```diff
@@ -0,0 +1,25 @@
+import type express from 'express'
+
+export const scopes = [
+  'demandA:read',
+  'demandA:create',
+  'demandA:comment',
+  'demandB:read',
+  'demandB:create',
+  'demandB:comment',
+  'match2:read',
+  'match2:propose',
+  'match2:cancel',
+  'match2:accept',
+  'match2:reject',
+] as const
+
+export type Scope = (typeof scopes)[number]
+
+export interface ScopedRequest extends express.Request {
+  user: {
+    jwt: {
+      scope?: string
+    }
+  }
+}
```

Comment:
it's probably worth moving this to a `global.d.ts` file and extend the express user instead of doing this https://github.com/digicatapult/sqnc-attachment-api/blob/main/src/global.d.ts

Reply:
done